rust/cyclotron-node/src/manager.ts

```diff
@@ -5,14 +5,15 @@ import { convertToInternalPoolConfig, serializeObject } from './helpers'
 import { CyclotronJobInit, CyclotronPoolConfig } from './types'
 
 export class CyclotronManager {
-    constructor(private config: { shards: CyclotronPoolConfig[] }) {
+    constructor(private config: { shards: CyclotronPoolConfig[], shardDepthLimit: number }) {
         this.config = config
     }
 
     async connect(): Promise<void> {
         return await cyclotron.maybeInitManager(
             JSON.stringify({
                 shards: this.config.shards.map((shard) => convertToInternalPoolConfig(shard)),
+                shard_depth_limit: this.config.shardDepthLimit,
```

Comment:
Where does this get read on the Rust side?

Reply:
It's an `Option<u64>` in the `ManagerConfig`, so serde should pick it up if it's set

Reply:
Ah, gotcha, I figured it would be added as part of this PR, didn't see it was there already. 👍